backend/src/main/java/com/bakdata/conquery/models/preproc/Preprocessed.java

```diff
@@ -254,6 +253,14 @@ public synchronized void addRow(int primaryId, PPColumn[] columns, Object[] outR
 		rowEntities.add(primaryId);
 
 		for (int col = 0; col < outRow.length; col++) {
+
+			if (values[col] == null && outRow[col] != null) {
+				throw new IllegalStateException(String.format("Expecting %s to be NULL, because no ColumnValues could be generated by his associated parser", outRow[col]));
```

Comment:
```suggestion
				throw new IllegalStateException(String.format("Expecting %s to be NULL, because no ColumnValues could be generated by the associated parser", outRow[col]));
```